auth/api/groups/http/requests.go

```diff
@@ -50,87 +49,102 @@ func (req updateGroupReq) validate() error {
 		return groups.ErrMalformedEntity
 	}
 
-	if req.ParentID != "" {
-		return groups.ErrParentInvariant
-	}
-
 	return nil
 }
 
 type listGroupsReq struct {
 	token    string
+	id       string
 	level    uint64
-	metadata groups.Metadata
-	name     string
-	groupID  string
-	tree     bool
+	tree     bool // If `true` result is JSON that represents groups hierarchy, otherwise JSON just holds array of groups.
```

Comment:
` If `true`, the response is a JSON that represents groups hierarchy, otherwise the JSON holds an array of groups.` something like that would be more understandable